src/components/Layout/ContactSection.js

```diff
@@ -79,68 +93,121 @@ const ContactSection = () => {
                                 <ListItem dense>
                                     <Button
                                         href="https://ni.fe.up.pt"
-                                        startIcon={<Language htmlColor={theme.palette.tertiary.main} />}
+                                        startIcon={
+                                            <Language
+                                                htmlColor={
+                                                    theme.palette.tertiary.main
+                                                }
+                                            />
+                                        }
                                         className={classes.colorText}
                                         title="website"
                                     >
-                                        <Typography variant="button">Website</Typography>
+                                        <Typography variant="button">
+                                            Website
+                                        </Typography>
                                     </Button>
                                 </ListItem>
                                 <ListItem>
                                     <Button
                                         href="https://facebook.com/NIAEFEUP/"
-                                        startIcon={<Facebook htmlColor={theme.palette.tertiary.main} />}
+                                        startIcon={
+                                            <Facebook
+                                                htmlColor={
+                                                    theme.palette.tertiary.main
+                                                }
+                                            />
+                                        }
                                         className={classes.colorText}
                                         title="facebook"
                                     >
-                                        <Typography variant="button">Facebook</Typography>
+                                        <Typography variant="button">
+                                            Facebook
+                                        </Typography>
```

Comment:
This is not a change request but there are lot of diffs caused by your auto formatter, it makes the review a bit harder to do. Maybe change its settings so it only triggers in your own changes?

Reply:
I found a setting for that, thanks :smiley_cat: